.github/dependabot.yml

```diff
@@ -16,7 +16,53 @@ updates:
         patterns:
           - "*"
 
+  # manual updates for npm dependencies that break the app or types
```

Comment:
I would write something here about why we want this second ecosystem, and add a link to the documentation about open PR limits.